dbms/src/Databases/DatabaseOrdinary.cpp

```diff
@@ -122,12 +122,12 @@ void DatabaseOrdinary::loadStoredObjects(
     FileNames file_names;
 
     size_t total_dictionaries = 0;
-    iterateMetadataFiles(context, [&file_names, &total_dictionaries, this](const String & file_name)
+    iterateMetadataFiles(context, [context, &file_names, &total_dictionaries, this](const String & file_name)
```

Comment:
Maybe &context to avoid copy?

Reply:
ok. I've never used C++ lambda functions.